src/main/java/com/dumbdogdiner/stickyapi/bukkit/book/commonmarkextensions/MCColorFormatDelimiterProcessor.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.book.commonmarkextensions;
+
+import com.dumbdogdiner.stickyapi.bukkit.book.chat.ComponentColor;
+import org.bukkit.ChatColor;
+import org.commonmark.node.*;
+import org.commonmark.parser.delimiter.DelimiterProcessor;
+import org.commonmark.parser.delimiter.DelimiterRun;
+
+import java.util.Iterator;
+import java.util.Locale;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.regex.Pattern;
+
+/**
+ * A delimiter processor for custom name nbt.
+ */
+public class MCColorFormatDelimiterProcessor implements DelimiterProcessor {
```

Comment:
Change this to `ColorFormatDelimiterProcessor`

```suggestion
public class MCColorFormatDelimiterProcessor implements DelimiterProcessor {
```